src/CountryPicker.js

```diff
@@ -130,15 +130,11 @@ export default class CountryPicker extends Component {
 
     setCountries(props.flagType)
     let countryList = [...props.countryList]
-    const excludeCountries = [...props.excludeCountries]
-
-    excludeCountries.forEach(excludeCountry => {
-      const index = countryList.indexOf(excludeCountry)
-
-      if (index !== -1) {
-        countryList.splice(index, 1)
-      }
-    })
+    const disabledCountries = [...props.disabledCountries]
+    this.disabledCountriesByCode = {};
+    disabledCountries.forEach(c => {
+      this.disabledCountriesByCode[c] = 1;
```

Comment:
```suggestion
      this.disabledCountriesByCode[c] = true;
```
I think it's better to use boolean here.